src/notify/event-renderers/renderers.ts

```diff
@@ -0,0 +1,11 @@
+import { notifyForEvent } from '../event'
+
+export const eventRenderers = new Map<string, (params: Parameters<typeof notifyForEvent>[0]) => Promise<void>>()
+export const registerEventRenderer = (
```

Comment:
@apexearth I think it is helpful to have some documentation on this part.

Like how you should use this function, what's expected and etc